templates/.snapshots/TestRenderDeploymentJsonnet-deployments-appname-app.jsonnet.tpl-deployments-testing-testing.jsonnet.snapshot

```diff
@@ -22,11 +22,19 @@ local all = {
 	namespace: ok.Namespace(app.namespace) {
 		metadata+: {
 			annotations+: {
-				'iam.amazonaws.com/permitted': '%s_service_role' % app.name,
+        'iam.amazonaws.com/permitted': '%s_service_role' % app.name,
```

Comment:
This indentation looks off. I imagine jsonnetfmt would fix it, but curious.

Reply:
Might be something having to do with tabs verse spaces